tools/cli/go-whisk-cli/commands/trigger.go

```diff
@@ -415,26 +421,30 @@ var triggerDeleteCmd = &cobra.Command{
 
```

Comment:
regarding lines
```
        // Get full feed name from trigger delete request as it is needed to delete the feed
        if retTrigger != nil && retTrigger.Annotations != nil {
```
what happens if retTrigger == nil or retTrigger.Annotations == nil (i.e. need to provide some `else` logic) ?